Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -24,6 +27,7 @@ QT_FORWARD_DECLARE_CLASS(QRadioButton)
 QT_FORWARD_DECLARE_CLASS(QScrollArea)
 QT_FORWARD_DECLARE_CLASS(QVBoxLayout)
 QT_FORWARD_DECLARE_CLASS(QStackedWidget)
+QT_FORWARD_DECLARE_CLASS(QDir)
```

Comment:
Note that when you include the actual header <Qdir> you should not forward declare the class - but it is preferable to not have the include in the header if possible and to use a forward declare so you can keep compile times optimized